adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -261,7 +376,12 @@ - (nullable UIView *)renderBannerAdInView:(nonnull UIView *)bannerView
                                    extras:(nullable VungleAdNetworkExtras *)extras
                            forPlacementID:(nonnull NSString *)placementID {
   NSError *bannerError = nil;
-  NSMutableDictionary *options = [[NSMutableDictionary alloc] init];  ///
+  NSMutableDictionary *options = [[NSMutableDictionary alloc] init];
+  if (extras.muted) {
```

Comment:
This line should be:

if (extras != nil)

The way it's written now, you're setting muted to YES every time.

Reply:
@ericleich Got it, I will fix this issue, thanks a lot for the guidance. 👍

Reply:
@ericleich  Done, thanks for your instruction. 😊